server/src/main/java/io/spine/server/model/declare/MethodSignature.java

```diff
@@ -119,12 +118,22 @@ public boolean matches(Method method) throws SignatureMismatchException {
         Collection<SignatureMismatch> mismatches = match(method);
         boolean hasErrors = mismatches.stream()
                                       .anyMatch(mismatch -> ERROR == mismatch.getSeverity());
+        List<SignatureMismatch> warnings =  mismatches.stream()
+                                                      .filter(MethodSignature::isWarning)
+                                                      .collect(toList());
         if (hasErrors) {
             throw new SignatureMismatchException(mismatches);
         }
+        if (!warnings.isEmpty()) {
+            warnings.stream().map(SignatureMismatch::toString).forEach(this::_warn);
+        }
         return true;
     }
 
+    private static boolean isWarning(SignatureMismatch mismatch){
+        return mismatch.getSeverity() == WARN;
```

Comment:
Looks like a method of `SignatureMismatch`. Together with `isError()`, this should enable us to delete `getSeverity()`.